src/assets/app.config.json

```diff
@@ -0,0 +1 @@
+{ }
```

Comment:
Do we need to have both files "app.config.example.json" and "app.config.json"? Why not only keeping the last one?

Reply:
When I will have written the documentation we can remove it, it is mostly to show case the feature.